pkg/balancer-js/src/pool-weighted/encoder.ts

```diff
@@ -89,4 +90,11 @@ export class WeightedPoolEncoder {
       ['uint256', 'uint256[]', 'uint256'],
       [WeightedPoolExitKind.BPT_IN_FOR_EXACT_TOKENS_OUT, amountsOut, maxBPTAmountIn]
     );
+
+  /**
+   * Encodes the userData parameter for exiting an InvestmentPool for withdrawing management fees.
+   * This can only be done by the pool owner.
+   */
+  static exitForManagementFees = (): string =>
```

Comment:
I'm not entirely sure if this should live here seeing as 99% of people aren't going to use it. At the very least I think it should probably be in an `InvestmentPoolEncoder`?

Reply:
Yes - this was just easier. If this design (method of extracting fees) survives, we can refactor

Reply:
Agreed for this initial release.